pkg/webhook/server.go

```diff
@@ -95,6 +95,10 @@ func (*Server) NeedLeaderElection() bool {
 	return false
 }
 
+func (*Server) GetID() string {
```

Comment:
Add a comment here and explain why we're returning an empty string

Reply:
added comment